src/Infrastructure/Persistence/Doctrine/Migrations/Version20230321170536.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230321170536 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE vehicle_characteristics DROP CONSTRAINT fk_54f8f40a9f073263');
+        $this->addSql('ALTER TABLE period DROP CONSTRAINT fk_c5b81ece9f073263');
+        $this->addSql('CREATE TABLE condition (uuid UUID NOT NULL, measure_uuid UUID NOT NULL, negate BOOLEAN NOT NULL, PRIMARY KEY(uuid))');
+        $this->addSql('CREATE INDEX IDX_BDD6884396A61612 ON condition (measure_uuid)');
+        $this->addSql('ALTER TABLE condition ADD CONSTRAINT FK_BDD6884396A61612 FOREIGN KEY (measure_uuid) REFERENCES measure (uuid) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE regulation_condition DROP CONSTRAINT fk_9d8762b796a61612');
+        $this->addSql('DROP TABLE regulation_condition');
+        $this->addSql('DROP INDEX uniq_c5b81ece9f073263');
+        $this->addSql('ALTER TABLE period RENAME COLUMN regulation_condition_uuid TO condition_uuid');
+        $this->addSql('ALTER TABLE period ADD CONSTRAINT FK_C5B81ECE638B0587 FOREIGN KEY (condition_uuid) REFERENCES condition (uuid) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_C5B81ECE638B0587 ON period (condition_uuid)');
+        $this->addSql('DROP INDEX uniq_54f8f40a9f073263');
+        $this->addSql('ALTER TABLE vehicle_characteristics RENAME COLUMN regulation_condition_uuid TO condition_uuid');
+        $this->addSql('ALTER TABLE vehicle_characteristics ADD CONSTRAINT FK_54F8F40A638B0587 FOREIGN KEY (condition_uuid) REFERENCES condition (uuid) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_54F8F40A638B0587 ON vehicle_characteristics (condition_uuid)');
```

Comment:
C'est probablement ce qu'a généré Doctrine, mais en faisant ça on perd à nouveau toutes les données

Pour les garder on peut directement renommer les objets, non ?

```suggestion
        $this->addSql('ALTER TABLE regulation_condition RENAME TO condition');
        $this->addSql('ALTER INDEX regulation_condition_pkey RENAME TO condition_pkey');
        $this->addSql('ALTER TABLE period RENAME COLUMN regulation_condition_uuid TO condition_uuid');
        $this->addSql('ALTER TABLE vehicle_characteristics RENAME COLUMN regulation_condition_uuid TO condition_uuid');
```

Reply:
Je me dis que la perte de données ici est "acceptable" dans le sens où nous n'avons que des données de tests, idem en production.

Reply:
Je suis un peu mal à l'aise avec cet argument quand éviter la perte de données est plutôt aisé

Dans la mesure du possible, autant se préparer dès aujourd'hui à gérer nos migrations comme si la perte de données était inacceptable (données réelles), non ?

Par ailleurs ici l'alternative "sans pertes" est sans doute plus simple

Reply:
> Que proposes-tu ? D'écrire la migration à la main, avec du rename table ?

Oui j'ai mis des suggestions en ce sens